Makefile

```diff
@@ -76,6 +76,13 @@ lint-golint:
 	@echo Running Go linter ...
 	@./hack/golangci-lint.sh run
 
+.PHONY: lint-codespell
+lint-codespell: CODESPELL_SKIP := $(shell cat .codespell.skip | tr \\n ',')
+lint-codespell: CODESPELL_BIN := codespell
```

Comment:
This line can be removed.

Reply:
@stevesloka commit `2985220` removes `lint-codespell: CODESPELL_BIN := codespell`. Should the same be done in projectcontour/contour?

Reply:
Yup! Looks like I missed it in the original PR but found it here. =) 

https://github.com/projectcontour/contour/pull/3141